packages/runner/src/lib/dom.js

```diff
@@ -71,6 +71,9 @@ function addElementBoxModelLayers ($el, $body) {
   const dimensions = getElementDimensions($el)
 
   const $container = $('<div class="__cypress-highlight">')
+  .css({
+    opacity: 0.7,
```

Comment:
Why the opacity change to a more opaque value? :D

Reply:
opacity used to be on each of the three elements, so it was really 0.6 stacked three times. So 0.7 looked the most correct to me